### DISCUSSION TITLE
Support logarithmic scale on Bar Chart axes

### DISCUSSION THREAD

**Discussion by xhejtman** [grafana/grafana#38500] [Dashboards, Panels & Visualizations]
Hello,

it seems that new bar chart (version 8.1.x) does not allow to set Y axis to log scale. Is this planned to add?

**Comment by dprokop** [comment]
We do not plan to add this any time soon, but we welcome contributions!

**Comment by pkolyvas** [comment]
@xhejtman I renamed the issue to make it a little easier for others to discover and contribute to. Can you tell us a little more about your workflow, use case, and why log scaling would help?

**Comment by xhejtman** [comment]
On the bar chart below, it would be better to use log scale instead of linear. It shows number of jobs (Y axis) and used CPUs (X axis)

![jobs](https://user-images.githubusercontent.com/22795052/130698222-8b4b632f-3cf5-4c76-bf92-2b4fa034ab77.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
